test/Unit/App/Attribute/ResourceDeprecationTest.php

```diff
@@ -46,4 +47,40 @@ public function testValidDatePassesValidation(): void
         $this->assertSame('test-link', $attribute->link);
         $this->assertSame('test-deprecation-reason', $attribute->deprecationReason);
     }
+
+    public function testToArray(): void
+    {
+        $class = new #[ResourceDeprecation(
+            sunset: '2038-01-01',
+            link: 'test-link',
+            deprecationReason: 'test-deprecation-reason',
+            rel: 'test-rel',
+            type: 'test-type',
+        )] class {
+        };
+
+        $reflectionClass = new ReflectionClass($class);
+        $attributes      = $reflectionClass->getAttributes(ResourceDeprecation::class);
+
+        $this->assertIsArray($attributes);
```

Comment:
```suggestion
        $this->assertNotEmpty($attributes);
```
psalm: RedundantConditionGivenDocblockType: Docblock-defined type list<ReflectionAttribute<Api\App\Attribute\ResourceDeprecation>> for $attributes is always array<array-key, mixed>